plaid/api_client.py

```diff
@@ -127,9 +129,11 @@ def __call_api(
         _return_http_data_only: typing.Optional[bool] = None,
         collection_formats: typing.Optional[typing.Dict[str, str]] = None,
         _preload_content: bool = True,
-        _request_timeout: typing.Optional[typing.Union[int, typing.Tuple]] = None,
+        _request_timeout: typing.Optional[typing.Union[int, float, typing.Tuple]] = None,
```

Comment:
Is request_timeout user configurable? If it is, this seems like it might be a breaking change?